api/routes/parseAllResume.js

```diff
@@ -0,0 +1,13 @@
+const saveAllResume = require("../services/saveAllResume/saveAllResume");
+
+const parseAllResume = async (req, res) => {
+  try {
+    const result = await saveAllResume();
+    res.json({ data: result });
```

Comment:
return smth?

Reply:
I don't know what function has to return